tests/test_integrated_channels/test_sap_success_factors/test_exporters/test_content_metadata.py

```diff
@@ -37,6 +37,87 @@ def setUp(self):
         self.addCleanup(jwt_builder.stop)
         super(TestSapSuccessFactorsContentMetadataExporter, self).setUp()
 
+    @ddt.data(
```

Comment:
Need to add a test case for a scenario when we don't have `first_enrollable_paid_seat_price` key on `course_runs`.